sql-plugin/src/main/scala/org/apache/spark/sql/catalyst/expressions/rapids/Timestamp.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.apache.spark.sql.catalyst.expressions.rapids
+
+import com.nvidia.spark.rapids.{ExprChecks, ExprRule, GpuExpression, GpuOverrides, TypeEnum, TypeSig}
+
+import org.apache.spark.sql.catalyst.expressions.{Expression, GetTimestamp}
+import org.apache.spark.sql.rapids.{GpuGetTimestamp, UnixTimeExprMeta}
+
+object TimeStamp {
+
+  def getExprs: Map[Class[_ <: Expression], ExprRule[_ <: Expression]] = Seq(
+    GpuOverrides.expr[GetTimestamp](
+      "Gets timestamps from strings using given pattern.",
+      ExprChecks.binaryProjectNotLambda(TypeSig.TIMESTAMP, TypeSig.TIMESTAMP,
+        ("timeExp", TypeSig.STRING, TypeSig.STRING),
```

Comment:
Actually Spark support more than just strings for `timeExp`, they also support Date and timestamp as input.

```
  override def inputTypes: Seq[AbstractDataType] =
    Seq(TypeCollection(StringType, DateType, TimestampType), StringType)
```

Reply:
@nartal1 you merged it too fast.  This needs to be fixed

Reply:
I am sorry @revans2, missed this.  Will put up a follow on PR now.